showcase/src/app/demo-ui/pages/dropdown/demo-dropdown-page.component.html

```diff
@@ -32,17 +38,19 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 		filesPath="dropdown/single-required-selection"
 		[exampleTitle]="'SHOWCASE.DEMO.DROPDOWN.REQUIRED' | translate"
 	>
-		<stark-dropdown
-			dropdownId="serviceDropdown"
-			[options]="serviceDropdownOptions"
-			[value]="selectedService"
-			placeholder="SHOWCASE.DEMO.DROPDOWN.SELECT_SERVICE"
-			optionIdProperty="id"
-			optionLabelProperty="value"
-			(selectionChanged)="serviceDropdownOnChange($event)"
-			required
-		>
-		</stark-dropdown>
+		<mat-form-field>
+			<stark-dropdown
+				dropdownId="serviceDropdown"
+				[options]="serviceDropdownOptions"
+				[value]="selectedService"
+				[placeholder]="'SHOWCASE.DEMO.DROPDOWN.SELECT_SERVICE' | translate"
+				optionIdProperty="id"
+				optionLabelProperty="value"
+				(selectionChanged)="serviceDropdownOnChange($event)"
+				required
```

Comment:
This example is not working as expected... although it is required, the dropdown is not marked in red when there is no value selected....

Reply:
Actually, this is working fine. Because there is no `ngControl` passed to the component, the `errorState` does not detect any error. Maybe we should add an `@Input` to let the developer passing his own `errorStateMatcher` as Angular for the mat-select

Reply:
Mmm in fact I think the issue is that in the `get errorState()` we just care about the `ngControl`... but we should also handle the case when there is no `ngControl`... in that case I think we can simply return something like `this.required && !this.value`